source/playgrounds.txt

```diff
@@ -30,12 +30,40 @@ files with the ``.mongodb`` extension as playgrounds.
 
 .. include:: /includes/admonitions/note-playgrounds.rst
 
-|vsce| splits your Playground window to display the results of your 
-Playground in |json| format in the right-side pane labeled ``Playground 
-Results.json``. If you disabled split-view, |vsce| displays the 
-Playground results in |json| format in a new tab to the right of 
-your Playground. You cannot edit the results in the results pane. 
-You can save the results to a file. 
+- If you enabled split-view, |vsce| splits your Playground window to
```

Comment:
This part is also a bit confusing. You can't enable or disable split-view. By default, it will display results in a split-view mode, but if you drag and drop your results window to a different position, it will be displayed as a tab. Not sure what you exactly mean here by enable/disable.

Reply:
It was here originally. I removed it.